src/interfaces/ServerCommands.ts

```diff
@@ -53,6 +53,10 @@ export const ServerCommands = {
    * worksheet.
    */
   NewScalaFile: "new-scala-file",
+  /**
+   * Create a worksheet with a default name in current directory
+   */
+  NewScalaWorksheet: "new-scala-worksheet",
```

Comment:
So I wasn't actually thinking of adding this in here. Mainly because technically it's not a server command right? Since it's actually just `new-scala-file` that we are using, which is the server command. I wasn't fully sure how editors would call this, and for example I actually already have this bound to `metals.quick-worksheet`, which I personally prefer, but the name doesn't matter. What do you think?

Reply:
Might be an overkill, I just thought that it might useful to have a common name, but it's not really needed in this case.

Reply:
Sounds good. I'll go ahead and release this now then so we can update the extensions.